app/dashboard/templates/profiles/profile.html

```diff
@@ -26,12 +26,30 @@
       {% endif %}
     </div>
     <div class="container-fluid bg-light pb-5">
-      {% if not hidden %}
+      </div>
         <div class="container profile-card">
           <div class="row">
             <div class="col-12 col-lg-6 profile-header__main-infos">
+              {% if not hidden %}
+              <div id="float_avatar_image" class="{% if is_my_profile %}is_my_profile{% endif %}">
+                {% if is_my_profile %}
+                  <a href="{% url "onboard" "profile" %}?steps=avatar">
+                {% endif %}
+                    <img class="profile-header__avatar mb-1 mr-4" src="{% if profile.avatar and profile.active_avatar.avatar_url %}{{ profile.active_avatar.avatar_url }}{% else %}{{ profile.avatar_url }}{% endif %}" />
+                {% if is_my_profile %}
+                    <div id="avatar_edit">
+                      <div id="avatar_edit_text">
+                        Edit Avatar
+                      </div>
+                    </div>
+                  </a>
+                {% endif %}
+              </div>
+
               <div class="clearfix">
```

Comment:
is this needed? I think can be removed

Reply:
no idea, not part of my code :/

Reply:
since now is an empty div can be removed without problem